server/src/controllers/taskController.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+const createTask = require('../services/createTask');
```

Comment:
For each service, it would be better to use name "....Service", for example, createTaskService etc